addons/binding/org.openhab.binding.freebox/README.md

```diff
@@ -0,0 +1,63 @@
+# Freebox Binding Configuration
+
+This binding integrates the [Freebox Revolution](http://www.free.fr/adsl/freebox-revolution.html) to your openHab installation.
+
+![Freebox Revolution](doc/freebox-revolution.jpg)
+
+##Binding configuration
+
+The binding will use the default address used by Free to access your Freebox Server (mafreebox.freebox.fr).
+
+## Authentication
+
+You'll have to authorise openHAB to connect to your Freebox. Here is the process described :
+
+**Step 1** At binding startup, if no token is recorded in the Item configuration, the following message
+will be displayed in the OSGi console :
+
+```
+            ####################################################################
+            # Please accept activation request directly on your freebox        #
+            # Once done, record current Apptoken in your 'openhab.cfg'         #

```

Comment:
there is actually no such thing as a openhab.cfg in openHAB 2! Instead, there should be a freebox.cfg file (content described in "Binding configuration" section.